pyintegration/deephaven2/plot/figure.py

```diff
@@ -162,42 +162,42 @@ def axes(
     def axis(
         self,
         dim: int = None,
-        t: Union[Table,SelectableDataSet] = None,
+        t: Union[Table, SelectableDataSet] = None,
         label: str = None,
-        color: Union[str,int,Color] = None,
+        color: Union[str, int, Color] = None,
         font: Font = None,
         font_family: str = None,
         font_size: int = None,
         font_style: str = None,
         format: AxisFormat = None,
         format_pattern: str = None,
-        min: Union[str,float] = None,
-        max: Union[str,float] = None,
+        min: Union[str, float] = None,
+        max: Union[str, float] = None,
         invert: bool = None,
         log: bool = None,
         business_time: bool = None,
-        calendar: Union[str,BusinessCalendar] = None,
+        calendar: Union[str, BusinessCalendar] = None,
         transform: AxisTransform = None,
     ) -> Figure:
         """TODO pydoc
```

Comment:
Same comment as before.  This looks complete for pydoc.  Is something still missing or is this done?